src/shared/components/curriculum-box/curriculum-box.css.ts

```
@@ -0,0 +1,30 @@
+import { themeVars } from '@/shared/styles/theme.css';
+import { typography } from '@/shared/styles/typography.css';
+import { style } from '@vanilla-extract/css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)